config/gatekeeper.yml

```diff
@@ -1603,7 +1603,6 @@ production:
     - 2P658W
     - 2L895T
     - 0V058D
-    - 0X337F
```

Comment:
This office code is now in the inactive code list so i have removed from the active list to avoid any possible future duplication if they were to be made active again